tests/baselines/reference/destructuringParameterDeclaration4.errors.txt

```diff
@@ -34,7 +36,11 @@ tests/cases/conformance/es6/destructuring/destructuringParameterDeclaration4.ts(
                 ~~~~~~~~~~~~~~~
 !!! error TS2370: A rest parameter must be of an array type.
     function a3(...b?) { }            // Error, can't be optional
+                    ~

```

Comment:
I checked the git history and they never had an error when they were added. Seems it has something to do with the other bogus baselines. They have always outputted the correct error in the compiler.